CadRevealComposer/SceneCreator.cs

```diff
@@ -154,7 +167,10 @@ private static SectorInfo SerializeSector(InternalSector p, string outputDirecto
     {
         var (estimatedTriangleCount, estimatedDrawCalls) = DrawCallEstimator.Estimate(p.Geometries);
 
-        var sectorFilename = p.Geometries.Any() ? $"sector_{p.SectorId}.glb" : null;
+        string? sectorFilename = !p.Geometries.Any()
+            ? null
+            : (!p.IsHighlightSector ? $"sector_{p.SectorId}.glb" : $"highlight_sector_{p.SectorId}.glb"); // The highlight_sector name is used in the frontend to identify the highlight sectors, don't rename without syncing with the frontend.
```

Comment:
Call the sectors something else? NOTE: Probably need to change the "guard" in the client (or handle it differently) before doing this